convert2rhel/pkgmanager/handlers/yum/__init__.py

```diff
@@ -163,6 +163,24 @@ def _enable_repos(self):
                 diagnosis="Loading repository metadata failed with error %s." % (str(e)),
             )
 
+    def _swap_problematic_packages(self):
+        """Swap system packages for their RHEL counterparts in the transaction.
+
+        Some packages need to be manually injected in the transaction as a
+        "swap", since those packages are not always able to be installed
+        automatically by yum if they don't exist in the system anymore, this
+        can cause problems during the transaction as missing dependencies.
+        Related issue: https://issues.redhat.com/browse/RHELC-1130, see comments
```

Comment:
I'm wondering if we should have the link to the issue in the docstring... Maybe I would move it to a comment inside the function, or try to give a better example/description inside it. I don't think that docstrings are a good place for putting that type of content in it. 

I don't have a strong opinion on that, so if others are comfortable, I'm okay with that as well.